qurro/tests/input/red_sea/README.txt

```diff
@@ -0,0 +1,3 @@
+These differentials were just computed by running Songbird normally. I haven't
+run any diagnostics to verify that Songbird was fitting things properly, so if
+you try this out yourself in Songbird you might get somewhat different results.
```

Comment:
Maybe point to the link in the songbird repo about using tensorboard to make reproducible results.